modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -292,13 +292,11 @@ public void testAddObjectAction() throws Exception {
 			WorkflowConstants.STATUS_APPROVED,
 			JSONUtil.getValue(
 				payloadJSONObject, "JSONObject/objectEntry", "Object/status"));
-		Assert.assertEquals(
-			"João",
+		Assert.assertNull(
 			JSONUtil.getValue(
 				payloadJSONObject, "JSONObject/objectEntry",
 				"JSONObject/values", "Object/firstName"));
-		Assert.assertEquals(
-			"João",
+		Assert.assertNull(
```

Comment:
@marcelabc I'm a little worried about this. Does this mean that `ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE` will no longer work like it used to?